index.html

```diff
@@ -109,6 +109,85 @@ <h4 class="features-about__title">
             </div>
         </div>
     </div>
+    <div class="pictures-wrapper">
+        <div class="pictures">
+            <div class="pictures-about">
+                <h5 class="pictures-about__title">Featured destinations</h5>
+                <button id="pictures-toggle" class="pictures-about-btn" type="button">
+                    View all
+                    <img class="pictures-about-btn__icon" src="./src/assets/image/Vector1.png">
+                </button>
+            </div>
+            <div class="pictures-column">
+                <div class="pictures-column-cart">
+                    <img class="pictures-column-cart__img" src="./src/assets/image/image1.png">
+                    <span class="pictures-column-cart__name">
+                        Raja Ampat
+                        <br> Indonesia
```

Comment:
Wrap Indonesia to the new line as it is in another blocks

Reply:
fixed